ui/lib/core/addon/components/form-field.hbs

```diff
@@ -9,7 +9,7 @@
       )
     )
   }}
-    {{#unless (eq @attr.type "object")}}
+    {{#unless (or (eq @attr.type "object") (eq @attr.options.editType "moreInfo"))}}
```

Comment:
Just a heads up that once the upgrade is merged the [simple-unless rule](https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/simple-unless.md) is set to error.